espei/utils.py

```diff
@@ -93,6 +93,8 @@ def optimal_parameters(trace_array, lnprob_array, kth=0):
     zeros. The number of chains and iterations in the trace and lnprob arrays
     must match.
     """
+    # Swap first two indices of trace_array for compatability with new version of emcee.
+    trace_array = np.swapaxes(trace_array, 0, 1)
```

Comment:
Thanks for being proactive on this. This is for the recently released `emcee>=3.0`, right? If that's the case, I'm glad to see that things are mostly working with updating.

I'm planning to try to phase out emcee once we have the likelihood gradient, so I'd prefer to pin the emcee version to `emcee<3.0` rather than try to migrate to 3.0 because accepting the new shape of the chain from emcee means that all downstream packages (e.g. [PDUQ](https://pduq.readthedocs.io/en/latest/)), examples and internal ESPEI in code have to change in several places.

Reply:
Ok, I undid the change. I'll just keep it on my local version though because I installed the newest version of emcee.